server.js

```diff
@@ -20,6 +20,85 @@ app.use(express.json());
 app.get("/", (req, res) => {
   res.send("Hello Technigo!");
 });
+////// Tuesday ////
+
+const { Schema } = mongoose;
+const HappyThoughtsSchema = new Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5, 
+    maxlength: 140
+  },
+  hearts: {
+    type: Number,
+    default: 0 
+
+  },
+  createdAt: {
+    type: Date,
+    default: new Date()
```

Comment:
Did you get this code to work? Impressed, I had to change it for it to log the time accordingly. Interesting!